infra/README.md

```
@@ -11,6 +11,86 @@ This directory contains all infrastructure components for the Mystira platform:
+│  │  • Virtual Networks, Subnets, NSGs                                    │  │
+│  │  • Azure AI, Key Vault, Service Bus                                   │  │
+│  │  • DNS zones, SSL certificates infrastructure                         │  │
+│  └───────────────────────────────────────────────────────────────────────┘  │
+│                                                                             │
+└─────────────────────────────────────────────────────────────────────────────┘
+```
+
+### Tool Comparison
+
+| Aspect | Terragrunt/Terraform | Harness GitOps |
+|--------|---------------------|----------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)